RecoBTag/FeatureTools/python/pfDeepDoubleXTagInfosNopt_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from pfDeepDoubleXTagInfos_cfi import pfDeepDoubleXTagInfos
+
+pfDeepDoubleXTagInfosNopt = pfDeepDoubleXTagInfos.clone(
+  min_jet_pt = cms.double(0)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.